DataCapturing/DataCapturingError.swift

```diff
@@ -44,4 +44,6 @@ public enum DataCapturingError: Error {
     case notRunning
     /// For some reason there was no current measurement to write data to or to read information to, during a capturing run. This can already happen during start up, if the current measurement was not created for some reason.
     case noCurrentMeasurement
+    /// The requested track does not exist
+    case invalidTrack
```

Comment:
An Exception currently only thrown by the STADTRADELN bridging code. If you like I could move it over there.